src/tiled/command.cpp

```diff
@@ -209,12 +216,20 @@ CommandProcess::CommandProcess(const Command &command, bool inTerminal)
 
     connect(this, SIGNAL(finished(int)), SLOT(deleteLater()));
 
+    if (showOutput)
+        connect(this, &QProcess::readyReadStandardOutput, this, &CommandProcess::consoleOutput);
+
     if (!mFinalWorkingDirectory.trimmed().isEmpty())
         setWorkingDirectory(mFinalWorkingDirectory);
 
     start(mFinalCommand);
 }
 
+void CommandProcess::consoleOutput()
+{
+    mLogger->log(LoggingInterface::INFO, QLatin1String(readAllStandardOutput()));
```

Comment:
You shouldn't wrap the returned `QByteArray` with a `QLatin1String`, cause that will break whenever the encoding of the output is not actually latin1. Since we can't really know the output encoding, we should probably assume local encoding, which means using [`QString::fromLocal8Bit`](https://doc.qt.io/qt-5/qstring.html#fromLocal8Bit-1).